packages/block-library/src/search/block.json

```
@@ -41,6 +41,10 @@
 	},
 	"supports": {
 		"align": [ "left", "center", "right" ],
+		"color": {
+			"gradient": true,
```

Just a typo here. The support flag for enabling gradients is actually plural.

```suggestion
			"gradients": true,
```